src/EnergyPlus/UnitarySystem.cc

```diff
@@ -14346,7 +14347,9 @@ namespace UnitarySystems {
                 }
             }
         } else if (SolFla == -2) {
-            PartLoadFrac = ReqOutput / FullOutput;
+            if (FullOutput != 0) {
+                PartLoadFrac = ReqOutput / FullOutput;
+            }
```

Comment:
There is likely a reason (a developer oversight) that FullOutput has not been calculated. Usually FullOutput is set near the beginning of this function when PLR is set to 1 and the system capacity is checked to see if that capacity exceeds the load, otherwise PLR = 1 and more calculations are not needed. One thing is for sure, if this function is iterating with SolveRoot then the system does have a non-zero FullOutput (or maybe the system is scheduled off? but I would think that would be caught early). To figure out the why you would need to step through this function at the time the error occurred.

Reply:
@rraustad I believe this catches the issue noted, but I'll hold this open for a little bit in case you want to take another quick look for any other higher level issues.